Android.bp

```diff
@@ -32,6 +32,8 @@ android_app {
         "androidx.lifecycle_lifecycle-livedata-ktx",
         "androidx-constraintlayout_constraintlayout",
         "com.google.android.material_material",
+        "setupcompat",
+        "setupdesign",
```

Comment:
Are these in AOSP?

Reply:
Yeah, I noted that in the commit message.

https://android.googlesource.com/platform/external/setupcompat/
https://android.googlesource.com/platform/external/setupdesign/